app/subscriber/src/features/home/utils/makeFilter.ts

```diff
@@ -23,6 +23,7 @@ export const makeFilter = (
     productIds: filter.productIds ?? undefined,
     publishedEndOn: filter.endDate ? filter.endDate : undefined,
     publishedStartOn: filter.startDate ? filter.startDate : undefined,
+    sentiment: filter.sentiment ?? undefined,
```

Comment:
Isn't filter an array or undefined?  This doesn't seem like a relevant operation.  Just use `filter.sentiment` I think.